src/basic/pidref.c

```diff
@@ -14,7 +14,28 @@
 #include "signal-util.h"
 #include "stat-util.h"
 
-bool pidref_equal(const PidRef *a, const PidRef *b) {
+int pidref_get_pidfd_id(PidRef *pidref) {
```

Comment:
this is a "set" more than a "get", given it doesn't return the id?

Reply:
Maybe "acquire"? As with `varlink_acquire_pidfd`.